### PR TITLE
Introduce Contentful::Importer namespace

### LINKED ISSUE #12: Refactor and namespace
Would you consider refactoring the code and using namespaces? Currently, there are several files under lib which could cause `require` issues or conflicts. It'd be great to see a more standardized ruby gem file structure and namespacing. I'd be happy to draft a version and do a pull request if you're open to it. Thanks!

### PR DESCRIPTION
**Pull request by neonichu**
There was some inconsistent existing namespacing going on, now there's just one module `Contentful::Importer`. Also moved library files into `lib/contentful/importer`.

Fixes #12

### CODE REVIEW

**Comment by neonichu** [comment]
Using <https://github.com/contentful/contentful-importer.rb/commit/69e1b2859db472194c1735c400e4d410ec619886?w=1> will make it a bit more feasible to see what has actually happened.

**Comment by pxlpnk** [comment]
Looks good. Can you update the changelog too?

File: lib/contentful/importer/converters/content_types_structure_creator.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Contentful
-  module Converter
+  module Importer
```
Comment (pxlpnk) [inline lib/contentful/importer/converters/content_types_structure_creator.rb]: Do we want to use the `Contentful::Importer::Converter` namespace here or is this too namespacey?

Comment (neonichu) [inline lib/contentful/importer/converters/content_types_structure_creator.rb]: I think it's too namespacey :)

Comment (pxlpnk) [inline lib/contentful/importer/converters/content_types_structure_creator.rb]: maximum over namespace engineering

**Pull request merged**